app/views/meeting.html

```diff
@@ -20,36 +20,39 @@
                 <!-- LOWER HEADER START  -->
                 <div class="col-md-1 ad-btn-group-right ad-lower-bg right ng-class:[{'col-sm-1':menuClosed(), 'col-sm-2':!menuClosed()}]">
                     <button type="button" class="btn btn-info ad-header-button" ng-click="ctrl.lowerClicked()">
-                        <i class="glyphicon center ng-class:[{'glyphicon-plus':!ctrl.isLowerMode(), 'glyphicon-minus':ctrl.isLowerMode()}]"></i>
+                        <i class="glyphicon center ng-class:[{'glyphicon-plus': !ctrl.isLowerMode(), 'glyphicon-minus': ctrl.isLowerMode()}]"></i>
                     </button>
                 </div>
                 <div class="col-xs-2 ad-btn-group-middle ad-lower-bg right">
-                    <button type="button" class="btn btn-primary ad-header-button" ng-click="ctrl.topicClicked()" translate translate-cloak>STR_PROPOSALS</button>
+                    <button type="button" class="btn btn-primary ad-header-button ng-class:[{'active': ctrl.isProposalsActive()}]" ng-click="ctrl.proposalsClicked()"
```

Comment:
I suppose a bit matter of taste whether to design by having a "isXXXActive()" or a generic function like the one ine `signitem.ctrl.js` which combines checks into one place so that they're not spread around.
I prefer the latter :)